src/Native/pkg/runtime.native.System.Security.Cryptography.Apple/runtime.native.System.Security.Cryptography.Apple.builds

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="runtime.native.System.Security.Cryptography.Apple.pkgproj"/>
+    <Project Include="osx\runtime.native.System.Security.Cryptography.Apple.pkgproj">
+      <OSGroup>osx.10</OSGroup>
```

Comment:
I just copied this value from the one for System.Security.Cryptography.Native.  It looks like all of the native packages use "osx.10"